src/main/java/dk/kb/util/YAML.java

```diff
@@ -25,10 +25,8 @@
 import java.net.MalformedURLException;
 import java.io.SequenceInputStream;
 import java.nio.file.InvalidPathException;
-import java.util.Arrays;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
* imports are evil. Configure your IDE to disallow them

Reply:
Agreed, will do.